### ISSUE TITLE
Error: Couldn't create [size] image [3DS]

### ISSUE THREAD

**Issue by minusplus** [EasyRPG/Player#2184]
#### Name of the game:
Most

#### Player platform:
3DS

#### Attach files (as a .zip archive or link them)
[easyrpg_log.txt](https://github.com/EasyRPG/Player/files/4558768/easyrpg_log.txt)

#### Describe the issue in detail and how to reproduce it:
Play a game for a while and eventually it will crash with the above error message. Loading a recent save will usually allow you to progress, unless there is a long period leading up to the error where you are unable to save, such as a cutscene.

**Comment by carstene1ns** [comment]
Duplicate of #1097.

**Comment by minusplus** [comment]
This is not a duplicate of that issue. That one describes long load times and an eventual crash in specific games. This issue describes a crash that happens in most games only after the game has been running for a while.

**Comment by carstene1ns** [comment]
Root cause is running out of memory.

**Comment by minusplus** [comment]
No it isn't. Root cause in the other issue is not enough memory to load the game. Root cause in this issue is not freeing up values in memory that are no longer needed, as indicated by being able to load a recent save and continue playing past the failure point.

**Comment by carstene1ns** [comment]
Then it is a duplicate of #1135, which we track inside #1097.

**Comment by minusplus** [comment]
Save04 isn't useful, but I'll attach it. I'm including Save05.lsd and another log as well.

[erpg.zip](https://github.com/EasyRPG/Player/files/4559265/erpg.zip)

If you load Save05 and go south, it will trigger a cutscene. I have consistently had it crash either before you have full control of your character again or shortly thereafter. Sometimes, during the battle, it throws up "Error allocating PNG pixel buffer" and fails to load animation frames, but continues playing. Sometimes it crashes with "Couldn't create [size] image" when loading a battle animation. Sometimes you get through the battle and another long cutscene and it crashes when you go south past the save point with "Couldn't create [size] image"

Save04 was made at that last save point. If you load Save04 you can continue past the teleport to the south without a crash. I don't know what happens after that point.
